### PR TITLE
prov/efa: fix a bug and enable multi client test

### PR DESCRIPTION
**Pull request by wzamazon**
Fix a bug in shm and enable the rdm_multi_client test.

### CODE REVIEW

File: prov/efa/src/rxr/rxr_init.c [hunk 1]
```diff
@@ -159,34 +160,44 @@ static void rxr_init_env(void)
...
- * Stringify the void *addr to a string smr_name formatted as `gid_qpn`, which
- * will be used to insert into shm provider's AV. Then shm uses smr_name as
- * ep_name to create the shared memory region.
+/* @brief convert raw address to an unique shm endpoint name (smr_name)
+ *
+ * Note even though all shm endpoints are on same instance. But because
+ * one instance can have multiple EFA device, it is still necessary
+ * to include GID on the name.
+ *
+ * a smr name is constisted of the following 4 parts:
```
Comment (zhngaj) [inline prov/efa/src/rxr/rxr_init.c:169]: NIT: consisted.

**Review by zhngaj — APPROVED** [review]
LGTM, just a typo.

File: prov/efa/src/rxr/rxr_init.c (hunk 1 above)
Comment (wzamazon) [reply prov/efa/src/rxr/rxr_init.c]: Thank you! fixed

**Comment by wzamazon** [comment]
bot:aws:retest

**Comment by wzamazon** [comment]
bot:aws:retest

**Pull request merged**